pages/common/q.md

```diff
@@ -0,0 +1,27 @@
+# q
+
+> Execute SQL-like queries on .csv and .tsv files.
+
+- Query .csv file by specifying the delimeter as ',':
+
+`q -d',' "SELECT * from {{path/to/file}}"`
+
+- Query .tsv file:
+
+`q -t "SELECT * from {{path/to/file}}"`
+
+- Query file with header row:
+
+`q -d{{delimiter}} -H "SELECT * from {{path/to/file}}"`
```

Comment:
It is not required. I did it in the first example for clarity.
I can update either example, I just wasn't sure which one was better.
`-d ','` (clarity) or `-d,` (brevity)